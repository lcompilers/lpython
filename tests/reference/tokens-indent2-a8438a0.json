{
    "basename": "tokens-indent2-a8438a0",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/indent2.py",
    "infile_hash": "19c2d8095cd65d521be708e23a2e29ab3cb672b888bc5fa8fbf745b3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-indent2-a8438a0.stderr",
    "stderr_hash": "ab862ff68983c5975d21f13c5b159de2a2888cae1d38a38a24700a88",
    "returncode": 1
}