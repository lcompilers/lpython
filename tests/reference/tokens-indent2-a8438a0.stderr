tokenizer error: Indentation should be of the same type (either tabs or spaces)
 --> tests/tokens/errors/indent2.py:4:1
  |
4 |  else:
  | ^ 
